### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by SoloSynth1**
- refactor nav structure
- rename directories and files for consistency
- delete unused placeholder file

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -30,7 +30,14 @@ theme:
...
+    - Toolchain:
+      - Ansible: "introduction/toolchain/ansible.md"
+      - Conda: "introduction/toolchain/conda.md"
   - 2. Installation:
-    - Overview: "./2-installation/OVERVIEW OF INSTALLATION PROCESS.md"
-    - Steps: "./2-installation/Installations step.md"
\ No newline at end of file
+    - Overview: "installation/OVERVIEW OF INSTALLATION PROCESS.md"
+    - Steps: "installation/Installations step.md"
+    - Installing miniconda: "installation/Installing Miniconda.md"
```
Comment (yamihe19) [inline mkdocs.yml]: is it Miniconda suppose to be Capitalize the first letter?

Comment (SoloSynth1) [reply mkdocs.yml]: Yes, from [their documentation site](https://docs.conda.io/en/latest/miniconda.html) it seems we need to capitalize it.

Comment (SoloSynth1) [reply mkdocs.yml]: Oh yes I see what you mean now, I will fix the un-capitalized page title.

Comment (SoloSynth1) [reply mkdocs.yml]: Fixed in [20d8d94](https://github.com/InTensorfiers/intensorfiers.github.io/pull/9/commits/20d8d94a88727d5850d590331258de4a24db6eda).

Comment (yamihe19) [reply mkdocs.yml]: Ok Thanks I will approved.

**Comment by yamihe19** [comment]
LGTM

**Review by yamihe19 — APPROVED** [review]

**Comment by yamihe19** [comment]
Great!

**Pull request merged**